game/outfits.rpy

```diff
@@ -1120,14 +1120,38 @@ init -1 python in jn_outfits:
 
     # Official JN eyewear
     __register_wearable(JNEyewear(
-        reference_name="jn_eyewear_circles",
-        display_name="Black circle glasses",
-        unlocked=True,
+        reference_name="jn_eyewear_round_glasses_black",
+        display_name="Black round glasses",
+        unlocked=False,
+        is_jn_wearable=True
+    ))
+    __register_wearable(JNEyewear(
+        reference_name="jn_eyewear_round_glasses_red",
+        display_name="Red round glasses",
+        unlocked=False,
+        is_jn_wearable=True
+    ))
+    __register_wearable(JNEyewear(
+        reference_name="jn_eyewear_round_glasses_brown",
+        display_name="Brown round glasses",
+        unlocked=False,
+        is_jn_wearable=True
+    ))
+    __register_wearable(JNEyewear(
+        reference_name="jn_eyewear_round_sunglasses",
+        display_name="Round sunglasses",
+        unlocked=False,
+        is_jn_wearable=True
+    ))
+    __register_wearable(JNEyewear(
+        reference_name="jn_eyewear_rectangular_glasses_black",
+        display_name="Black rectangular glasses",
+        unlocked=False,
         is_jn_wearable=True
     ))
     __register_wearable(JNEyewear(
-        reference_name="jn_eyewear_heart_frames",
-        display_name="Pink heart glasses",
```

Comment:
Do you plan to delete them totally or should I tune the pink a bit down and we can use them later?

Reply:
Good point - I'll leave that to your best judgment

Reply:
Did we have an idea of where to go with this?

Reply:
If you want you can delete them from the quote. We still have the files on drive in case we want to bring them back we can.